config/path

```diff
@@ -45,6 +45,9 @@ TARGET_PREFIX=$TOOLCHAIN/bin/$TARGET_NAME-
 # use ARM toolchain on 64/32 split builds
 if [ -z "$KERNEL_TOOLCHAIN" -a "$TARGET_KERNEL_ARCH" = "arm64" -a "$TARGET_ARCH" = "arm" ]; then
   KERNEL_TOOLCHAIN="aarch64-none-linux-gnu"
+  if [ "$(uname -m)" = "aarch64" ]; then
```

Comment:
You can use `${MACHINE_HARDWARE_NAME}` in place of `$(uname -m)`.

Reply:
Addressed. Thanks

Reply:
You could use `${MACHINE_HARDWARE_NAME}` in `openssl/package.mk`, too. Originally I didn't think we were setting `${MACHINE_HARDWARE_NAME}` "unconditionally" but upon re-reading the code in `config/optimize` it should always be set - we just avoid (re-)setting it when it is already set.